resources/statements.ts

```diff
@@ -59,3 +59,18 @@ export interface StatementsListParams {
      */
     sort?: string
 }
+
+
+export class GetStatementRequest {
+    public statementId: string
+    public outputType: string
+    public customerId: string | undefined
+    public language: string
+
+    constructor(statementId: string, outputType: string = "html", language: string = "en", customerId?: string) {
```

Comment:
Please change `outputType` and `language` form string to new types containing their possible values.
especially because we are using  `outputType` to determine to which route to send the request